src/pp.js

```diff
@@ -51,8 +51,8 @@ function conditionalGroup(states, opts) {
   );
 }
 
-function ifBreak(contents) {
-  return { type: "if-break", contents };
+function ifBreak(breakContents, flatContents) {
+  return { type: "if-break", breakContents, flatContents };
```

Comment:
I forgot, you should do this here:

```
assertDoc(breakContents);
assertDoc(flatContents);
```

This gives much nicer errors when you do something wrong, because where you passed in a bad value will be exactly in the stack trace.